src/lib/discord/commands/admin.ts

```diff
@@ -17,52 +17,22 @@ import type {
   InteractionReplyOptions,
   ThreadChannel,
 } from 'discord.js'
+import type { Question } from '@prisma/client'
 
 const userIdToUsername = new Map<string, User>()
 
-/** @TODO replace with colors endpoint once this is deployed */
 const iconMap = new Map<string, string>([
-  [
-    'Admin',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Admin.svg',
-  ],
-  ['Bot', 'https://raw.githubusercontent.com/esauerbo1/Images/main/Bot.svg'],
-  [
-    'Amplify Bot Dev',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Amplify-Bot-Dev.svg',
-  ],
-  [
-    'Moderator',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Moderator.svg',
-  ],
-  [
-    'Amplify Staff',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Amplify-Staff.svg',
-  ],
-  [
-    'AWS Staff',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/AWS-Staff.svg',
-  ],
-  [
-    'Community Builder',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Community-Builder.svg',
-  ],
-  [
-    'Contributor',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Contributor.svg',
-  ],
-  [
-    'Meetup Organizer',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Meetup-Organizer.svg',
-  ],
-  [
-    'Amplify Guru',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/Amplify-Guru.svg',
-  ],
-  [
-    'at-everyone',
-    'https://raw.githubusercontent.com/esauerbo1/Images/main/%40everyone.svg',
-  ],
+  ['Admin', 'FD0061'],
```

Comment:
these color codes should appear on the Discord roles from the API (and on the messages in the interaction), can these be loaded dynamically?